tracing/tests/support/field.rs

```diff
@@ -19,8 +19,9 @@ pub struct MockField {
     value: MockValue,
 }
 
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Debug, PartialEq)]
 pub enum MockValue {
+    F64(f64),
```

Comment:
minor nit, but since the test support code now handles `f64` values, we should probably also add some tests that actually try to record `f64`s...

Reply:
Where would be the best place to put these? I looked for example in the tracing-core/src/field.rs but most of the tests there were calling `record_debug` directly.

Reply:
I would probably put them in https://github.com/tokio-rs/tracing/blob/master/tracing/tests/event.rs and https://github.com/tokio-rs/tracing/blob/master/tracing/tests/span.rs